src/interfacers/EmonHubMinimalModbusInterfacer.py

```diff
@@ -223,6 +253,11 @@ def read(self):
 
 
     def set(self, **kwargs):
+
+        def as_list(src):
```

Comment:
This is because a single value in the meter field appears to be read as individual chars, i.e.
              registers = 724
comes out as if
              registers = 7,2,4